src/components/Label.vue

```diff
@@ -1,7 +1,16 @@
 <script setup>
-defineProps({ label: String });
+defineProps({
+  label: String,
+  sticky: Boolean
+});
 </script>
 
 <template>
-  <h4 class="eyebrow border-b py-2 px-4 text-skin-text" v-text="label" />
+  <h4
+    class="eyebrow border-b py-2 px-4 text-skin-text"
+    :class="{
+      'sticky top-[71px] bg-skin-bg': sticky
```

Comment:
I think it should be 72px here, otherwise if you are on the settings page the "Voting" tab will move from 1px when you scroll
```suggestion
      'sticky top-[72px] bg-skin-bg': sticky
```